### DISCUSSION TITLE
How about a popup menu for the tabs?

### DISCUSSION THREAD

**Discussion by edwinyzh** [sumatrapdfreader/sumatrapdf#2350] [Ideas]
How about add a context menu for the tabs with the following menu items? (or at least add to the File main menu):

- Close All Tabs (must have)
- Close Other Tabs (nice to have)

Thanks.

**Comment by GitHubRulesOK** [comment]
covered in 
https://github.com/sumatrapdfreader/sumatrapdf/issues/371
https://github.com/sumatrapdfreader/sumatrapdf/issues/713

Workaround is use Ctrl+N to open new window then close first
or use Ctrl+Shift+N then close other window(s)
